### PR TITLE
More webhook events

### PR DESCRIPTION
**Pull request by charvp**
This PR adds automatic comments for pushes, branch or tag creation and deletion, issue and pull request close events, issue comments, pull request reviews and pull request review comments.

### CODE REVIEW

**Comment by charvp** [comment]
Github won't allow me to upload pictures right now so hosted on imgur:

https://i.imgur.com/aU4cxmy.png

**Comment by charvp** [comment]
This fixes #26, by the way.

**Comment by jwilander** [comment]
Awesome, I'll take a look sometime this weekend

File: server/command.go [hunk 1]
```diff
@@ -90,7 +95,7 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*mo
 
 	switch action {
 	case "subscribe":
-		features := "pulls,issues"
+		features := "pulls,issues,pushes,creates,deletes,issue_comments,pull_reviews"
```
Comment (jwilander) [inline server/command.go]: I wonder if we should default to less, do you think this might be too spammy by default?

Comment (hanzei) [reply server/command.go]: I would just set it to `"pulls,issues,creates,deletes". `issue_comments` and `pull_reviews` are way to spammy. `pushes` are debatable.

File: server/command.go [hunk 2]
```diff
@@ -17,10 +17,15 @@ const COMMAND_HELP = `* |/github connect| - Connect your Mattermost account to y
...
+    * issues - includes new and closed issues
+	* pulls - includes new and closed pull requests
+    * pushes - includes pushes
+    * creates - includes branch and tag creations
+    * deletes - includes branch and tag deletions
+    * issue_comments - includes new issue comments
+    * pull_reviews - includes pull request reviews
 	* label:"<labelname>" - must include "pulls" or "issues" in feature list when using a label
-  * Defaults to "pulls,issues"
+  * Defaults to "pulls,issues,pushes,creates,deletes,issue_comments,pull_reviews"
```
Comment (jwilander) [inline server/command.go]: Can you update the help text too?

Comment (charvp) [reply server/command.go]: Right, that's why i shouldn't code in the evening :smile:

Comment (jwilander) [reply server/command.go]: 😅

**Comment by jwilander** [comment]
Thanks @charvp !

**Pull request merged**